cpp/include/cudf/reduction.hpp

```diff
@@ -23,13 +23,16 @@
  * @brief These enums indicate the supported reduction operations that can be
  * performed on a column
  */
-typedef enum {
-  GDF_REDUCTION_SUM = 0,        ///< Computes the sum of all values in the column
-  GDF_REDUCTION_MIN,            ///< Computes the minimum of all values in the column
-  GDF_REDUCTION_MAX,            ///< Computes the maximum of all values in the column
-  GDF_REDUCTION_PRODUCT,        ///< Computes the multiplicative product of all values in the column
-  GDF_REDUCTION_SUMOFSQUARES,   ///< Computes the sum of squares of the values in the column
-} gdf_reduction_op;
+enum gdf_reduction_op {
```

Comment:
Put this in the `reductions` namespace and eliminate the prefix
```suggestion
enum operators {
```

Reply:
`operators` is very generic (Note: not inside namespace) and might have name clash in future. Can I rename to `reduction_op` or `reduction_operators` ?

Reply:
Let's move it into the `reductions` namespace to avoid the clash.